### DISCUSSION TITLE
Gerätekommunikation gestört

### DISCUSSION THREAD

**Discussion by ThreeSixty123** [OpenCCU/OpenCCU#2727] [🇩🇪 Fragen&Antworten]
Guten Tag,
nachdem ich die heutige Umstellung meines Raspberry Pies auf SSD vorgenommen habe, habe ich mich dazu entschlossen alles von Grund auf neu zu erstellen.
Nach Installation des HA habe ich HACS und dann Raspberry Matic instlliert.

Fix die Thermostate und Fenster Sensoren angelegt und  dann im Anschluss auch in Gruppen zusammen gefügt.

1	Arbeitszimmer	HmIP-Heizungssteuerung	INT0000001	Arbeitszimmer INT0000001	
2	Schlafzimmer	HmIP-Heizungssteuerung	INT0000002	Schlafzimmer INT0000002	
3	Wohnzimmer	HmIP-Heizungssteuerung	INT0000003	Wohnzimmer INT0000003	
4	Badezimmer	HmIP-Heizungssteuerung	INT0000004	Badezimmer INT0000004

Das ging 3 Stunden gut. Nun wollte ich die Wochenprogramme einstellen und auf einmal passiert es, dass ich Servicemeldungen bekomme. Auch mit den HMIP-Geräten welche sich im selben Raum befinden gibt es Probleme.

Dabei ist es egal ob ich über Homeassistant die Wochenprogramme "Boost" "Keine" "WP1" "WP" oder "WP3" auswählen. Immer geht die Verbindung verloren. 
Irgendwann kommt sie dann wieder, oder aber auch nicht.

Sobald ich die Einstellungen bei "INT" an die Gruppe übertragen möchte bekomme ich "Gerätekommunikation gestört" für das Thermostat (HmIP-eTRV-2) und die Gruppe (INT0000001-4)

Carriersense = 0% & Duty Cycle auf 9%

Es ist immer der Erste Kanal welcher nicht mehr Verfügbar ist , also der mit der XXX:0 am Ende

Auch das löschen der Gruppen und Ansteuern einzelner Geräte gibt die Fehlermeldung: Gerätekommunikation gestört

Hier mal eine Fehlermeldung:

> Dieser Fehler wurde von einer benutzerdefinierten Integration verursacht
> 
> Logger: hahomematic.client
> Quelle: custom_components/homematicip_local/climate.py:295
> Integration: Homematic(IP) Local (Dokumentation, Probleme)
> Erstmals aufgetreten: 15:32:37 (5 Vorkommnisse)
> Zuletzt protokolliert: 15:47:26
> 
> SET_VALUE failed with ClientException [XMLRPC Fault from backend: -1 Generic error (TIMEOUT)]: 000A17098F4884:1, ACTIVE_PROFILE, 2
> SET_VALUE failed with ClientException [XMLRPC Fault from backend: -1 Generic error (TIMEOUT)]: 000A18A9AA1794:1, BOOST_MODE, False
> SET_VALUE failed with ClientException [XMLRPC Fault from backend: -1 Generic error (TIMEOUT)]: 000A1A499B95EC:1, ACTIVE_PROFILE, 2
> SET_VALUE failed with ClientException [XMLRPC Fault from backend: -1 Generic error (TIMEOUT)]: 000A18A9AA1794:1, ACTIVE_PROFILE, 1
> SET_VALUE failed with ClientException [XMLRPC Fault from backend: -1 Generic error (TIMEOUT)]: 000A17098F4884:1, BOOST_MODE, False

Da es ja eine ganz frische Installation ist weiß ich nicht mehr weiter.

Habt ihr eine Hilfe?

**Comment by Baxxy13** [comment]
> Raspberry Pies auf SSD

Das ist zu generisch. 
Was für ein RaspberryPi? 
Was für eine SSD?
Wie/wo ist die SSD angeschlossen?
Was für ein Funkmodul und wie/wo angeschlossen?

Das ganze klingt irgendwie nach Funkstörungen.

**Comment by ThreeSixty123** [comment]
Hi, 
lange ging es gut nun heute wieder massive Störungen welche lange andauern.

> -raspberrymatic user.info monit[1361]: Starting Monit 5.33.0 daemon with http interface at /var/run/monit.sock
> Sep 13 08:23:04 de838cd8-raspberrymatic user.info monit[1361]: 'de838cd8-raspberrymatic' Monit 5.33.0 started
> Sep 13 08:23:19 de838cd8-raspberrymatic user.err monit[1361]: 'udevdEnabled' status failed (0) -- no output
> Sep 13 08:23:20 de838cd8-raspberrymatic user.err monit[1361]: 'sshdEnabled' status failed (1) -- no output
> Sep 13 08:23:20 de838cd8-raspberrymatic user.err monit[1361]: 'hs485dEnabled' status failed (2) -- grep: /var/etc/hs485d.conf: No such file or directory
> Sep 13 08:23:20 de838cd8-raspberrymatic user.err monit[1361]: 'hmlangwEnabled' status failed (1) -- no output
> Sep 13 08:23:20 de838cd8-raspberrymatic user.warn monit[1361]: 'hasUSB' status failed (1) -- no output
> Sep 13 08:23:20 de838cd8-raspberrymatic user.err monit[1361]: 'hb_rf_eth-CheckEnabled' status failed (1) -- no output
> Sep 13 08:23:20 de838cd8-raspberrymatic user.warn monit[1361]: 'rpi4usb3Check' status failed (1) -- no output
> Sep 13 08:23:20 de838cd8-raspberrymatic user.err monit[1361]: 'tailscaleEnabled' status failed (1) -- no output
> Sep 13 08:23:20 de838cd8-raspberrymatic user.err monit[1361]: Lookup for '/media/usb1' filesystem failed  -- not found in /proc/self/mounts
> Sep 13 08:23:20 de838cd8-raspberrymatic user.err monit[1361]: Filesystem '/media/usb1' not mounted
> Sep 13 08:23:20 de838cd8-raspberrymatic user.err monit[1361]: 'usb1' unable to read filesystem '/media/usb1' state
> Sep 13 08:23:20 de838cd8-raspberrymatic user.info monit[1361]: 'usb1' trying to restart
> Sep 13 08:23:36 de838cd8-raspberrymatic user.warn monit[1361]: 'hasUSB' status failed (1) -- no output
> Sep 13 08:23:36 de838cd8-raspberrymatic user.warn monit[1361]: 'rpi4usb3Check' status failed (1) -- no output
> Sep 13 08:23:36 de838cd8-raspberrymatic user.err monit[1361]: Filesystem '/media/usb1' not mounted
> Sep 13 08:23:36 de838cd8-raspberrymatic user.err monit[1361]: 'usb1' unable to read filesystem '/media/usb1' state
> Sep 13 08:23:36 de838cd8-raspberrymatic user.info monit[1361]: 'usb1' trying to restart
> Sep 13 08:23:46 de838cd8-raspberrymatic local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"0000DBE99AD798:1","STATE"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: STATE"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2618]
> Sep 13 08:23:46 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"0000DBE99AD798:1","STATE"}) [CallGetValue():iseXmlRpc.cpp:1445]
> Sep 13 08:23:46 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]


Das ist zu generisch.
Was für ein RaspberryPi? Raspberry Pi 4 b 
Was für eine SSD? Samsung Cruacial SSD 850 
Wie/wo ist die SSD angeschlossen? An einem USB 2 Port ( USB 3 soll Probleme machen )
Was für ein Funkmodul und wie/wo angeschlossen? 
Über einen USB HUB an USB 2:
/dev/ttyACM0
usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2494718-if00 
aber irgendwie sehe ich den Homematic IP RF-USB-Stick  :(

**Comment by Peet1985** [comment]
Ich habe das selbe Problem, nur dass sich an meinem System seit 5 Jahren nichts geändert hat ausser die Firmware auf der CCU3. Aktuell habe ich die 3.77.7.20240826 drauf.

Auch werden aktualierungen nicht an die CCU übergeben. Ich habe zb. auf dem Wandtermostat bereits 23,4°C stehen aber in der CCU steht noch 29°C. Erst wenn ich irgendein Befehl an das Termostat sende bekomme ich die aktuelle Temperatur.

**Comment by ThreeSixty123** [comment]
Naja bei mir kommt nirgendwo mehr was an. 

So gleich ist das nicht. 

Geräte blinken orange und am Ende einmal rot wenn der Status sich geändert hat wie z. B. Fenster auf. 

Ich habe so langsam das Gefühl das durch Firmware Updates der PI4 mittlerweile auch Probleme mit USB 2 hat in Kombi mit einem usb hub. Denn die USB 3 hab ich ja schon gar nicht mehr in Benutzung

**Comment by Peet1985** [comment]
Doch das hier angesproche Problem habe ich auch. Das von mir beschriebene Problem, habe ich zusätzlich beobachtet.

Wenn ich eine Temperatur ändere, bricht ab und zu die Geräteverbindung zur CCU ab (Entsprechende Servicemeldung wird auch ausgegeben). Wenn ich dann die Temperatur am Termostat ändere, kommt das Gerät zurück. Es funktioniert allerdings auch, die Temperatur über die CCU neu zu senden, da kommt das Gerät auch zurück.
